MSBX5420-teamproject-vt.R

```diff
@@ -123,3 +123,117 @@ usmap <- usa +
 
 ani.options(interval = 0.5)
 gganimate(usmap)
+
+
+#Create new column to enter shelter-in-place-dates for each city, enter 1 (binary value) for shelter-in-place date
+us_no2_2020$shelter_in_place <- ifelse(us_no2_2020$Date>"2020-03-20" & us_no2_2020$City=="Chicago", 1, 
+                                       ifelse(us_no2_2020$Date>"2020-03-20" & us_no2_2020$City == "Los Angeles", 1,
+                                              ifelse(us_no2_2020$Date>"2020-03-20" & us_no2_2020$City == "Manhattan", 1,
+                                                     ifelse(us_no2_2020$Date>"2020-03-18" & us_no2_2020$City == "San Francisco", 1,
+                                                            ifelse(us_no2_2020$Date>"2020-03-23" & us_no2_2020$City == "Seattle", 1,
+                                                                   ifelse(us_no2_2020$Date>"2020-04-01" & us_no2_2020$City == "Washington D.C.", 1, 0))))))

```

Comment:
Logical test modified to be ">" instead of "==" date so we have a running list of ones for rows occurring in a time window after stay at home order goes into effect for that location.